tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalServerBehavior.java

```diff
@@ -847,12 +790,123 @@ private void setupAriesJmxBundles()
         }
     }
 
+    @Override
```

Comment:
Don't move this to a new location int eh file, it makes it harder to see the git diff.